src/plugins/Aria.js

```diff
@@ -38,22 +38,24 @@ export default class Aria {
   }
 
   /**
-   * Create a CustomEvent.
+   * Dispatch a CustomEvent.
    *
    * @static
    * @param {String} type The type of the event.
    * @param {Object} detail Data to be passed with the event.
-   * @returns {Object}
+   * @param {HTMLElement} detail The element upon which the event should be dispatched.
    */
-  static createAriaEvent(type, detail = {}) {
-    return new CustomEvent(
+  static dispatchAriaEvent(type, detail = {}, element) {
+    const event = new CustomEvent(
```

Comment:
This appears to create a new `CustomEvent` every time you want to dispatch, even if that event is identical to events created before. Is that your intention and, if so, is that a best practice?

Reply:
Hmmm this is a great question. That's the intention, though I'm not sure about best practice. The event for each method likely wouldn't change, since for example the `ariaHide` event will always have `expanded: false` in the `details` object (or at least is _should_), so this could probably be cleaned up and/or simplified quite a bit. Thanks!